packages/tools/src/enums/JumpPresets.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Jump Presets - This enum defines the 3 jump states which are available
+ * to be used with the jumpToSlice utility function.
+ */
+enum JumpPresets {
```

Comment:
Hmm, I was asumming this would be at the app level, this seems like a rather opinionated thing to have at the library level, when camera position/imageIndex covers all basis and is more generic?